content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
+MBCFOPM6JW2... wasmcloud.azurecr.io/echo:0.1.0                                         
+                                
+Provider ID    Link Name        Image Reference                        
+VDHPKGFKDI3... default          N/A                                    
+VAHNM37G4AR... __wasmcloud_...  N/A                                    
+VAG3QITQQ2O... default          wasmcloud.azurecr.io/httpserver:0.10.0 
+```
 
-# Call to action is default enabled, if you want to disable it. just change the 
-enable = false
+#### Linking Actors and Capability Providers
+Now your `Echo` Actor and `HTTP Server` Providers are running, but they aren't connected. In order to allow the Actor and Provider to communicate, we need to `link` them together, which we do using their respective Public Keys (shown in the Inventory as `Actor ID` and `Provider ID`).
 
-####### Default parameters ###########
-[params]
-logo = "images/logo.png"
-# Meta data
-description = "This is meta description"
-author = "Themefisher"
-# contact form action
-contact_form_action = "#" # contact form works with https://formspree.io
-# Google Analitycs
-google_analitycs_id = "" # your id
+You can use the following command to `link` your Actor and Provider, on MacOS you might receive a message asking if you want `wash` to receive incoming network connections. You should click "Allow", this is just setting up the provider's local HTTP server to listen on port 8080.
+```shell
+wash> ctl link MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M wasmcloud:httpserver PORT=8080
+```
+Once you see that the link has been advertised, you are ready to send a request to your actor.
 
-```
\ No newline at end of file
+#### Interacting with your Actor
+In another terminal window (not the REPL), run the following command:
+```shell
+curl localhost:8080/echo
+```
+In response, you should receive your request object (notice the path argument):
+```shell
+{"method":"GET","path":"/echo","query_string":"","headers":{"host":"localhost:8080","user-agent":"curl/7.64.1","accept":"*/*"},"body":[]}
+```
+Feel free to try out different methods of making a request to your Actor, including adding headers or using a different HTTP method to see different outputs.
+
+Instead of using `curl`, you can also _directly invoke_ actors' registered functions using `ctl call`. As mentioned before, the function that "echoes" this HTTP request is registered with the operation `HandleRequest`, and we can make this request directly to the actor if we supply the correct parameters.
+
+Here's an example of using `call` in the REPL to mimic our above `curl`. Note that this is not interacting with the HTTP server provider, and we don't need it to be running or linked for this operation to succeed:
+```shell
+wash> ctl call MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 HandleRequest {"method": "GET", "path": "/echo", "body": "", "queryString":"","header":{}}
+```

```

Comment:
Might want to use a caution/warning emoji here and drop a call-out explaining that this only works because the JSON payload you're passing is _exactly_ the same in terms of fields, shape, and data types as the messagepack payload the actor is expecting. If they don't match this up correctly, the actor will reject the call.